Memmy-App/memmy
# fix #779: add missing translations

### PR Creator Checklist

Ensure you've checked the following before submitting your PR:


### Summary

- fixes #779
- fix dayjs locales updating (non-lazy-load solution for now)
- cover missing translations (mostly)

### Screenshots

_n/a_

### Test Plan

1. Go to issue #779
2. Check the images of missing translations
3. Check the app for missing translations in english/german (obviously I cannot check the other languages)
